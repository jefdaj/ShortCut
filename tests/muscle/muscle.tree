.
|-- cache
|   |-- load
|   |   |-- 82645ed6f6.faa
|   |   |-- 82645ed6f6.faa.ids
|   |   |-- a810086108.faa
|   |   `-- a810086108.faa.ids
|   `-- makeblastdb
|       `-- e00fb07ddb
|           |-- e00fb07ddb.pdb.phr
|           |-- e00fb07ddb.pdb.pin
|           `-- e00fb07ddb.pdb.psq
|-- exprs
|   |-- blastp_db
|   |   `-- 4f7795b38a_4d0e43d616_46b5852d89.bht
|   |-- extract_seqs
|   |   `-- c08cc19988_86fa47f1b5.faa
|   |-- extract_targets
|   |   `-- 84cae9923d.str.list -> ../../cache/lines/...
|   |-- list
|   |   `-- fa4677789c.faa.list -> ../../cache/lines/...
|   |-- load_faa
|   |   |-- 33c746245c.faa -> ../../cache/load/82645ed6f6.faa
|   |   `-- 387f499ddc.faa -> ../../cache/load/a810086108.faa
|   |-- makeblastdb_prot
|   |   `-- ced061f0b2.pdb -> ../../cache/lines/...
|   |-- muscle
|   |   `-- a8af1a2fe0.aln
|   |-- num
|   |   `-- efc8362dcc.num -> ../../cache/lines/...
|   `-- str
|       |-- 1d8a61c770.str -> ../../cache/lines/...
|       `-- b9eaf59b0c.str -> ../../cache/lines/...
`-- vars
    |-- hits.bht -> ../exprs/blastp_db/4f7795b38a_4d0e43d616_46b5852d89.bht
    |-- hits_aln.aln -> ../exprs/muscle/a8af1a2fe0.aln
    |-- hits_faa.faa -> ../exprs/extract_seqs/c08cc19988_86fa47f1b5.faa
    |-- pcc7942.faa -> ../exprs/load_faa/387f499ddc.faa
    |-- query.faa -> ../exprs/load_faa/33c746245c.faa
    `-- result -> ../vars/hits_aln.aln

15 directories, 24 files
