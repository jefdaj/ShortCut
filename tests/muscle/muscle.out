CLUSTAL W (1.81) multiple sequence alignment

Synpcc7942_0071      MCSSSECPVTSIPAFNLTEQYRRIGAELETATAKVLASGGYIGGQTVADFEAAFA...
Synpcc7942_0002      --------------------------MAMSDQPEELRVSDLLNRQVL---DRQTT...
Synpcc7942_2044      -----------------------------MTAEQVWLRSEIMGTQVI---TRDSG...
...

